tests-legacy/resources/modules/statsbestsuppliers/statsbestsuppliers.php

```diff
@@ -0,0 +1,156 @@
+<?php
+/*
+* 2007-2015 PrestaShop
+*
+* NOTICE OF LICENSE
+*
+* This source file is subject to the Academic Free License (AFL 3.0)
+* that is bundled with this package in the file LICENSE.txt.
+* It is also available through the world-wide-web at this URL:
+* http://opensource.org/licenses/afl-3.0.php
+* If you did not receive a copy of the license and are unable to
+* obtain it through the world-wide-web, please send an email
+* to license@prestashop.com so we can send you a copy immediately.
+*
+* DISCLAIMER
+*
+* Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+* versions in the future. If you wish to customize PrestaShop for your
+* needs please refer to http://www.prestashop.com for more information.
+*
+*  @author PrestaShop SA <contact@prestashop.com>
+*  @copyright  2007-2015 PrestaShop SA
+*  @license    http://opensource.org/licenses/afl-3.0.php  Academic Free License (AFL 3.0)
+*  International Registered Trademark & Property of PrestaShop SA
+*/
+
+if (!defined('_PS_VERSION_')) {
+    exit;
+}
+
+class statsbestsuppliers extends ModuleGrid
+{
+    private $html = null;
+    private $query = null;
+    private $columns = null;
+    private $default_sort_column = null;
+    private $default_sort_direction = null;
+    private $empty_message = null;
+    private $paging_message = null;
+
+    public function __construct()
+    {
+        $this->name = 'statsbestsuppliers';
+        $this->tab = 'analytics_stats';
+        $this->version = '2.0.0';
+        $this->author = 'PrestaShop';
+        $this->need_instance = 0;
+
+        parent::__construct();
+
+        $this->default_sort_column = 'sales';
+        $this->default_sort_direction = 'DESC';
+        $this->empty_message = $this->trans('Empty recordset returned', array(), 'Admin.Notifications.Info');
+        $this->paging_message = $this->trans('Displaying %1$s of %2$s', array('{0} - {1}', '{2}'), 'Admin.Global');
+
+        $this->columns = array(
+            array(
+                'id' => 'name',
+                'header' => $this->trans('Name', array(), 'Admin.Global'),
+                'dataIndex' => 'name',
+                'align' => 'center'
+            ),
+            array(
+                'id' => 'quantity',
+                'header' => $this->trans('Quantity sold', array(), 'Admin.Global'),
+                'dataIndex' => 'quantity',
+                'align' => 'center'
+            ),
+            array(
+                'id' => 'sales',
+                'header' => $this->trans('Total paid', array(), 'Modules.Statsbestsuppliers.Admin'),
+                'dataIndex' => 'sales',
+                'align' => 'center'
+            )
+        );
+
+        $this->displayName = $this->trans('Best suppliers', array(), 'Modules.Statsbestsuppliers.Admin');
+        $this->description = $this->trans('Adds a list of the best suppliers to the Stats dashboard.', array(), 'Modules.Statsbestsuppliers.Admin');
```

Comment:
```suggestion
        $this->description = $this->trans('Enrich your stats, add a list of the best suppliers to the dashboard.', array(), 'Modules.Statsbestsuppliers.Admin');
```

Reply:
Don't waste your time on wording in this PR, we are only playing with tests :sweat_smile: